.github/workflows/old-compilers.yml

```
@@ -86,47 +88,55 @@ jobs:
             BUILD_TYPE: Release
             COMPILER: clang
             VERSION: 14
+            STATS: OFF
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)